Source/JavaScriptCore/API/glib/JSCClass.cpp

```diff
@@ -477,9 +477,9 @@ GRefPtr<JSCClass> jscClassCreate(JSCContext* context, const char* name, JSCClass
     GUniquePtr<char> prototypeName(g_strdup_printf("%sPrototype", priv->name.data()));
     JSClassDefinition prototypeDefinition = kJSClassDefinitionEmpty;
     prototypeDefinition.className = prototypeName.get();
-    JSClassRef prototypeClass = JSClassCreate(&prototypeDefinition);
-    priv->prototype = jscContextGetOrCreateJSWrapper(context, prototypeClass);
-    JSClassRelease(prototypeClass);
+    RefPtr prototypeClass = JSClassCreate(&prototypeDefinition);
+    priv->prototype = jscContextGetOrCreateJSWrapper(context, prototypeClass.get());
+    JSClassRelease(prototypeClass.get());
```

Comment:
Don't release it manually. Use adoptRef().

Reply:
JSClassRef is not a GObject, but a C API object. so I guess you mean to use JSRetainPtr

Reply:
Ah!, I see you wrote adoptRef, not adoptGRef, so it's indeed right